app-kotlin/build.gradle

```diff
@@ -1,47 +1,95 @@
 buildscript {
   repositories {
-    jcenter()
+    mavenCentral()
     google()
   }
 
   dependencies {
-    classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.0'
+    classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10'
   }
 }
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'org.jetbrains.kotlin.android'
+
 
 android {
-  compileSdkVersion 31
+  compileSdkVersion 33
 
   defaultConfig {
     applicationId 'au.com.gridstone.trainingkotlin'
     minSdkVersion 21
-    targetSdkVersion 31
+    targetSdkVersion 33
     versionCode 1
     versionName '1.0'
   }
 
   buildTypes {
     release {
       minifyEnabled false
-      proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
     }
   }
 
-  compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
-  }
-  kotlinOptions {
-    jvmTarget = '1.8'
+  sourceSets {
+    main.java.srcDirs += 'src/main/kotlin'
   }
 }
 
 dependencies {
-  implementation 'androidx.core:core-ktx:1.7.0'
+  implementation 'org.jetbrains.kotlin:kotlin-stdlib:1.7.0'
```

Comment:
do we need this?

Reply:
yeah we don't need this cleaning  thanks